kafka-rest/src/main/java/io/confluent/kafkarest/controllers/BrokerConfigManagerImpl.java

```diff
@@ -48,28 +47,35 @@
   @Override
   public CompletableFuture<Map<Integer, List<BrokerConfig>>> listAllBrokerConfigs(
       String clusterId, List<Integer> brokerIds) {
-    List<ConfigResource> brokerResources = brokerIds.stream()
+    List<ConfigResource> brokerResources =
+        brokerIds.stream()
             .map(brokerId -> new ConfigResource(Type.BROKER, String.valueOf(brokerId)))
             .collect(Collectors.toList());
     return listConfigs(
-        clusterId,
-        brokerResources,
-        BrokerConfig.builder().setClusterId(clusterId).setBrokerId(-1))
-        .thenApply(configs -> configs.entrySet().stream().collect(Collectors.toMap(
-            e -> Integer.parseInt(e.getKey().name()),
-            e -> e.getValue().stream().map(config -> BrokerConfig.create(
-                    config.getClusterId(),
-                    Integer.parseInt(e.getKey().name()),
-                    config.getName(),
-                    config.getValue(),
-                    config.isDefault(),
-                    config.isReadOnly(),
-                    config.isSensitive(),
-                    config.getSource(),
-                    config.getSynonyms()
-              )).collect(Collectors.toList())
-            )
-        ));
+            clusterId,
+            brokerResources,
+            BrokerConfig.builder().setClusterId(clusterId).setBrokerId(-1))
+        .thenApply(
```

Comment:
This one looks easier to read before the changes.

Reply:
I think here the new formatting actually exacerbates the underlying problem, which is that this lambda->lambda->lambda is too nested. Ugly things should look ugly :)